src/index.html

```diff
@@ -4,6 +4,13 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Simply Chocolate</title>
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="./index.css" />
     <script type="module" src="./index.js"></script>
```

Comment:
Din conspect am inteles ca daca adaugam scripturile in Head, browserul va rula toate scripturile pana la final, inainte sa incarce altceva. Ar trebui sa le adaugam la finalul codului html.